ansible/roles/seldon/defaults/main.yaml

```diff
@@ -8,9 +8,83 @@ seldon_install_runtime: true
 seldon_install_servers: true
 
 seldon_force_install_crds: false
+seldon_force_install_components: false
```

Comment:
In other Ansible playbooks we tend to use a single flag
```yaml
helm_force_install: false
```
that is used for every Helm installation - unless we need to be more granular potentially single flag is easier?

Reply:
In the `seldon` role here (ansible for core v2) there was only one "force" option available, namely `seldon_force_install_crds`. For the others, no option was available, and nothing was being passed to the corresponding task. 

I typically just force the components when doing incremental deployments (i.e running the ansible playbook over an existing kind cluster), hence the finer granularity.

But I can set them all by default to a common`helm_force_install` and let the dev choose

Reply:
hmm.... what we also tend to have in other place and may be useful here are flags to disable installation of all components except one, we have for example 
```yaml
full_install: yes

install_istio: "{{ full_install }}"
install_nginx: "{{ full_install }}"
install_knative_eventing: "{{ full_install }}"
install_knative_serving: "{{ full_install }}"

...
```

and one can do `-e full_install=no -e install_istio=yes`

Reply:
This is the same here. This PR retains all the normal core v2 ansible install features (which has those options) and just adds additional config capabilities via an extra config vars file. 

In fact, providing some of those `install_...` options within this extra vars file helps express everything declaratively in one place. For example, an extra vars file that spins up a cluster connected to Confluent Kafka would also have `install_kafka: false` inside so as to not use strimzi.

Reply:
Ah yes, it is just `full_install` being defined in entrypoint playbook and not deeper in the role like in other place, indeed...

Then single `helm_force_install` should suffice and help overall keep number of variables lower

Reply:
I've added the `helm_force_install` top-level option. The individual `seldon_force...` variables now default to its value (similar to the `full_install` case)